openreview/conference/templates/programchairWebfield.js

```diff
@@ -847,7 +848,7 @@ var calcMetaReviewCount = function(blindedNotes) {
 var calcDecisions = function(blindedNotes) {
   var finalDecisions = [];
   blindedNotes.forEach(function(note) {
-    if (note.details.decision) {
+    if (note.details.decision && note.details.decision !== 'No Decision') {
```

Comment:
```suggestion
    if (note.details.decision) {
```

Reply:
thank it's reading a wrong property here but we can't remove this
removing this condition will cause the decision progress data in overview tab to be wrong where "No Decision" is considered as a final decision and the decision progress data will always be 100%